great_expectations/cli/datasource.py

```diff
@@ -680,14 +681,29 @@ def create_expectation_suite(
         batch_kwargs=batch_kwargs,
         profiler=profiler,
         expectation_suite_name=expectation_suite_name,
-        run_id=datetime.datetime.now().isoformat().replace(":", "") + "Z",
+        run_id=run_id,
         additional_batch_kwargs=additional_batch_kwargs
     )
 
     if profiling_results['success']:
         build_docs(context)
         if open_docs:  # This is mostly to keep tests from spawning windows
-            context.open_data_docs()
+            from great_expectations.data_context.types import DataAssetIdentifier, ExpectationSuiteIdentifier, \
+                ValidationResultIdentifier
```

Comment:
Are these imports deferred to avoid a circular dependency? I noticed earlier that we should do some more thoughtful reorganization of so-called "types" imports; if that's the case, I'd like to flag this for further follow-up.

Reply:
I moved these imports into the top of the file and it seems to be working now.